src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        final String formatString = "dd.MM.yyyy";
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(formatString);
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info): If you create a [formatter](https://docs.oracle.com/javase/tutorial/datetime/iso/format.html), make it a constant field.